app/src/main/java/org/feature/fox/coffee_counter/data/local/database/dao/UserDao.kt

```diff
@@ -41,6 +50,10 @@ interface UserDao {
     @Query("SELECT * FROM purchase WHERE userId=:id")
     suspend fun getPurchaseListOfUser(id: String): List<Purchase>
 
+    @Transaction
+    @Query("SELECT * FROM image WHERE userId=:id")
+    suspend fun getImageById(id: String): Image
+
```

Comment:
Is there any reason, why you're using the `@Transaction` annotation?

Reply:
Solved in [2af8ace](https://github.com/Morphclue/code-camp-2/pull/98/commits/2af8ace824426ffb1bfa9ba262cda5582e958247).